themes/gokarna/static/css/main.css

```diff
@@ -12,18 +13,27 @@ body {
   margin-bottom: 170px;
 }
 
+a:hover {
+  color: #0058b5;
```

Comment:
This is the same as the color used in tags

Reply:
Cool